tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -262,6 +262,12 @@ void parse_seconds() {
 
     fail_parse("1.2 1.3", "%S %S", time_ms);
     fail_parse("1.2 2.2", "%S %S", time_ms);
+
+    // LWG-3536, failed parse leaves duration unmodified.
+    minutes mm{20};

```

Comment:
Just for a sanity check, I'd love a few additional test cases.  The case that you have is great for checking invalidity due to truncation (e.g., the given type would need to truncate the information parsed because it cannot store all of it) but it would also be great to have test cases for
- category mismatch (e.g., parse a month and try to store it in a duration)
- an obviously invalid input string (e.g., `""` and try to store the parse result in a duration)
  - Thanks @MattStephanson for the suggestion!